docs/source/index.rst

```diff
@@ -21,13 +21,19 @@ apps by DPF and their related format:
      - DPF examples
    * - MAPDL
      - .rst, .mode, .rfrq, .rdsp
-     - :ref:`_ref_distributed_msup`, :ref:`_ref_average_across_bodies`
+     - :ref:`distributed_msup`, :ref:`average_across_bodies`
    * - LS DYNA
      - .d3plot
      - :ref:`lsdyna_operators`
-   * - Fluent
-     - .cfx, .cas/dat.h5, .res, .flprj
-     - :ref:`_ref_fluids_mesh`
+   * - Fluent CFF restart files
+     - .cas/dat.h5
+     - :ref:`fluids_results`
+   * - CFX CFF files
+     - .cas/dat.cff
+     - :ref:`fluids_mesh`
+   * - Fluid project files
+     - .flprj
+     - :ref:`fluids_mesh`
```

Comment:
@MichaelNale I think that now that we have specific examples, it makes more sense to point there. In particular, the first example gives an overview of the MeshInfo and ResultInfo